R/data.R

```diff
@@ -52,3 +52,68 @@
 #' @source Compiled by Alessandro Chiarenza. See item descriptions for details.
 "GTS2012"
 
+#' Early tetrapod data from the Paleobiology Database
+#'
+#' A dataset of tetrapod occurrences ranging from the Carboniferous through to
+#' the Early Triassic. Data from: \url{https://paleobiodb.org/}.
+#' Dataset includes a range of variables relating to identification, geography,
+#' environmental context, traits and more. Those relevant to common paleobiological
+#' analyses are described below, but more information (and information on other
+#' variables) can be found at \url{https://paleobiodb.org/data1.2/}.
+#'
+#' @format A data frame with 5270 rows and 73 variables:
```

Comment:
I wonder if we really need so many variables to show off the functionality of the package? Remember that we need to keep everything under 5 MB so I think we should dump a load of these columns that probably aren't needed? Currently this file is almost 400 KB, so it is consuming almost 10% of this allowance.

Reply:
I offered you a version with less variables and you said the full, raw one was better :) but sure, I'll take a few of the super unnecessary ones out. We can also use a smaller clade (Amphibia, Reptilia) if we run out of space later.